Tic-Tac-Toe with AI (Java)/task/src/tictactoe/play/HardMode.java

```diff
@@ -22,21 +18,22 @@ void addItemsToLookupTable() {
 	@Override
 	void computerInput() {
 		GameStatus gameStatus = new GameStatus();
-		addItemsToLookupTable();
-
 		gameStatus.setTicTacToe(ticTacToeTable);
-		char turn = gameStatus.turn() ? 'X': 'O';
-		char opposingTurn = gameStatus.turn() ? 'O': 'X';
+		int bestScore = (int) Double.NEGATIVE_INFINITY;
```

Comment:
instead of  Double.NEGATIVE_INFINITY just use -1 
and for Double.POSITIVE_INFINITY  just use 1
 because we are not updating any value in score